app/src/main/res/layout/onboarding_finish.xml

```diff
@@ -2,11 +2,35 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<Button xmlns:android="http://schemas.android.com/apk/res/android"
-    style="@style/PositiveButton"
-    android:id="@+id/finish_button"
-    android:background="@drawable/onboarding_padded_background"
-    android:backgroundTint="?actionPrimary"
-    android:layout_marginBottom="10dp"
-    android:layout_marginHorizontal="@dimen/home_item_horizontal_margin"
-    android:text="@string/onboarding_finish" />
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/conclusion"
+    android:layout_marginTop="32dp"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginHorizontal="@dimen/home_item_horizontal_margin">
+
+    <TextView
+        android:id="@+id/header_text"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:textAppearance="@style/Header20TextStyle"
+        tools:text="@string/onboarding_conclusion_header"
+        android:letterSpacing="0.05"
```

Comment:
Where is this `letterSpacing` coming from? I think we can generally ignore the letterSpacing from the figma.